ui/components/metadata.js

```diff
@@ -0,0 +1,12 @@
+export default function Metadata({ data }) {
+  return (
+    <div className='flex flex-col space-y-2 pt-3'>
```

Comment:
I'd recommend using `grid` here

Reply:
Check out https://github.com/infrahq/infra/blob/main/ui/pages/providers/index.js#L52 for an example!